current/standard-derivatives.md

```diff
@@ -67,9 +66,10 @@ TODO
 How do I extract binary information of PDFs, audio files, video files, word processor files, spreadsheet files, presentation program files, and text files to a CSV file, or into the [Apache Parquet](https://parquet.apache.org/) format to [work with later](df-results.md#what-to-do-with-dataframe-results)?
```

Comment:
"How do I extract binary information " --> "How do I extract **the** binary information"